toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -237,3 +237,5 @@ python3-libs-3.7.7-10.cm1.aarch64.rpm
 python3-setuptools-3.7.7-10.cm1.noarch.rpm
 python3-xml-3.7.7-10.cm1.aarch64.rpm
 systemd-rpm-macros-239-37.cm1.noarch.rpm
+which-2.21-7.cm1.aarch64.rpm
+which-debuginfo-2.21-7.cm1.aarch64.rpm
```

Comment:
I don't think we need the debug information.

Reply:
Fxied.